docs/release_process.md

```diff
@@ -0,0 +1,319 @@
+    gitGraph
+       commit
+       branch change-1
+       commit
+       commit
+       checkout main
+       merge change-1
+```
+
+This means many features could be in development at the same time, and all can
+merge back to `main` when they are ready.
+
+```mermaid
+%%{init: { 'theme': 'base', 'gitGraph': {'rotateCommitLabel': true} } }%%
+    gitGraph
+       commit
+       branch change-1
+       commit
+       commit
+       checkout main
+       branch change-2
+       commit
+       checkout main
+       merge change-1
+       branch change-3
+       commit
+       commit
+       checkout main
+       merge change-2
+       checkout change-3
+       commit
+       commit
+       checkout main
+```
+
+## Release Timeline
+The standard release interval for Monitor is 1 week, meaning every week there
+will be a new version of the Monitor web app on the [Production][prod]
+environment. To do this, we first release code to [Dev][dev] and
+[Stage][stage].
+
+## Release to Dev
+Every commit to `main` is automatically deployed to the [Dev][dev] server, as
+long as it can be done with a fast-forward push. Since the
+[Great GitHub Heroku Incident of 2022][github-heroku-incident], this is
+done from CircleCI using a [service account][service-account].
+
+To push a different branch, you need to add the Heroku app as a remote:
+
+* `heroku login`
+* `heroku git:remote -a fx-breach-alerts`
+
+Then, you can push your local unmerged branch to Heroku:
+
+* `git push -f heroku change-1:main`
+
+Merges to main will fail to deploy until someone manually resets it to `main`:
+
+* `git push -f heroku main`
+
+
+## Release to Stage
+Every tag pushed to GitHub is automatically deployed to the [Stage][stage]
+server. The standard practice is to create a tag from `main` every Tuesday at
+the end of the day, and to name the tag with `YYYY-MM-DD` [CalVer][calver]
+syntax. This tag will include only the changes that have been merged to `main`.
+E.g.,
+
+1. `git tag 2022.08.02`
+2. `git push --tags`
+
+E.g., the following `2022.08.02` tag includes only `change-1` and `change-2`.
+```mermaid
+%%{init: { 'theme': 'base', 'gitGraph': {'rotateCommitLabel': true} } }%%
+    gitGraph
+       commit
+       branch change-1
+       commit
+       commit
+       checkout main
+       branch change-2
+       commit
+       checkout main
+       merge change-1
+       branch change-3
+       commit
+       commit
+       checkout main
+       merge change-2 tag: "2022.08.02"
+       checkout change-3
+       commit
+       commit
+       checkout main
+```
+
+### Create Release Notes on GitHub
+After you push the tag to GitHub, you should also
+[make a pre-release on GitHub][github-new-release] for the tag.
+
+1. Choose the tag you just pushed (e.g., `2022.08.02`)
+2. Type the same tag name for the releae title (e.g., `2022.08.02`)
+3. Click "Previous tag:" and choose the tag currently on production.
+   * You can find this at [the `__version__` endpoint][prod-version].
```

Comment:
We should restore/add this some time!

Reply:
Do we have a Jira ticket for this? I didn't find anything obvious in GitHub or Jira.

Reply:
This is on the `nextjs` branch now! https://github.com/mozilla/blurts-server/commit/374eab444d9490cc61a443a2c7babea013e6af0e